### DISCUSSION TITLE
Please make "stopped the pause stream" error clearer and more specific

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#8498] [General]
Non technical users will not understand what that means

Even for technical users it is not clear enough _What_ stream is stopped?

QUIC or HTTP/2?

One scenario is the server closed the connection without sending any data in case of NGINX (444)

it is better to say the server closed the connection without headers

**Comment by jay** [comment]
That should probably be shown only when DEBUG_HTTP2 is defined. It is the stream that returned NGHTTP2_ERR_PAUSE to nghttp2.
~~~diff
diff --git a/lib/http2.c b/lib/http2.c
index b8c9773..ef3fb4b 100644
--- a/lib/http2.c
+++ b/lib/http2.c
@@ -1236,7 +1236,7 @@ void Curl_http2_done(struct Curl_easy *data, bool premature)
       (void)nghttp2_session_send(httpc->h2);
 
     if(http->stream_id == httpc->pause_stream_id) {
-      infof(data, "stopped the pause stream!");
+      H2BUGF(infof(data, "stopped the pause stream!"));
       httpc->pause_stream_id = 0;
     }
   }
~~~

**Reply by bagder** [reply]
I agree, it is mostly (only?) usable for us who debug/develop the nghttp2 code within curl.

**Reply by bagder** [reply]
See #8502
